pkg/kubelet/images/image_manager_test.go

```diff
@@ -241,17 +244,22 @@ func pullerTestEnv(t *testing.T, c pullerTestCase, serialized bool, maxParallelI
 	fakeRuntime = &ctest.FakeRuntime{T: t}
 	fakeRecorder := &record.FakeRecorder{}
 
-	fakeRuntime.ImageList = []Image{{ID: "present_image:latest"}}
+	fakeRuntime.ImageList = []kubecontainer.Image{{ID: "present_image:latest"}}
 	fakeRuntime.Err = c.pullerErr
 	fakeRuntime.InspectErr = c.inspectErr
 
 	fakePodPullingTimeRecorder = &mockPodPullingTimeRecorder{}
+	tmpStateDir, err := os.MkdirTemp("", "image_manager_state")
```

Comment:
Would it be possible to use `testing.T.TempDir`? Then you don't need error checking and the directory will be removed automatically for you.

Reply:
Good catch. Updated.